SourceKitStressTester/Tests/StressTesterToolTests/ActionGeneratorTests.swift

```diff
@@ -50,6 +50,105 @@ class ActionGeneratorTests: XCTestCase {
     """)
   }
 
+  func testCompletionExpectedResults() {
+    typealias TestCase = (source: String, expected: [String])
+    let cases: [TestCase] = [
+      ("foo(x: 10)", ["call:foo(x:)"]),
+      ("foo(45)", ["call:foo(_:)"]),
+      ("foo.bar(first, x: 10)", ["foo", "call:bar(_:x:)", "first", "x:"]),
+      ("foo.bar(x:_:)", ["foo", "bar(x:_:)"]),
+
+      // FIXME: code completion doesn't complete compound name references
+      (".foo(x:y:z:)", ["foo(x:y:z:)"/*, "x", "y", "z"*/]),
+      (".foo(1, 2, other: 5)", ["call:foo(_:_:other:)", "other:"]),
+      ("Foo.foo(fooInstance)(x: 10)", ["Foo", "call:foo(x:)", "fooInstance"]),
+
+      // The below foo could be either foo(_:) or foo(_:_:), we can't tell
+      // syntactically so default to the one with fewer args as that will still
+      // the longer one if that's what it ends up being in the completion
+      // results (the extra param will be assumed to be defaulted).
+      ("Foo.foo(fooInstance)(10, 20)", ["Foo", "call:foo(_:_:)", "fooInstance"]),
```

Comment:
The above comment does not match the test case. In the comment you are arguing we should require the one with fewer arguments but the test case looks for the call with two arguments.